platform/solr/solr-schema/src/main/resources/solr/conf/solrconfig.xml

```diff
@@ -1303,6 +1304,32 @@
     </arr>
   </requestHandler>
 
+  <searchComponent name="gazetteer" class="solr.SuggestComponent">
+    <lst name="suggester">
+      <str name="name">gazetteerSuggest</str>
+      <str name="lookupImpl">BlendedInfixLookupFactory</str>
+      <str name="indexPath">gazetteer_index</str>
+      <str name="dictionaryImpl">DocumentDictionaryFactory</str>
+      <str name="suggestAnalyzerFieldType">lowercase</str>
+      <str name="field">name_txt</str>
+      <str name="weightField">sort-value_int</str>
+      <str name="sortField">population_lng</str>
+      <str name="payloadField">id_txt</str>
+      <str name="buildOnStartup">true</str>
```

Comment:
❗ This has been an problem in production before since it can delay how quickly Solr nodes can recover from restarts.  It should be faster with the smaller focused collection but it is probably safer to continue to decouple the suggester rebuilds from Solr node restarts.

>Some lookup implementations may take a long time to build, especially with large indexes. In such cases, using buildOnCommit or buildOnOptimize, particularly with a high frequency of softCommits is not recommended; it’s recommended instead to build the suggester at a lower frequency by manually issuing requests with suggest.build=true.
https://lucene.apache.org/solr/guide/8_5/suggester.html#suggester-search-component-parameters

Reply:
Should I go back to trying to build async when the bundle starts up? 
Just for reference with a 70k catalog it only takes on the order of seconds (maybe up to 7 seconds)? This single line was nice because it replaced something like 30 lines of code that retried until it was up and then built it.  (and if we don't build when we startup then the client throws errors when it attempts to be used)

Reply:
I think production could have 10+ million at a minimum if not more and growing.

Do we need that first async build?  Or can we just rely on the commands?

Reply:
Yeah we can rely on the commands , just gotta make sure its built before people use it or it throws errors and breaks the UI 😅  Maybe thats a sign we need a ui fix for this 😄

Reply:
Does sound like we need to make the UI handle failures better.

Reply:
set to false